src/adapters/MeetingsJSONAdapter.js

```diff
@@ -403,16 +407,22 @@ export default class MeetingsJSONAdapter extends MeetingsAdapter {
 
       deviceId = availableDevices.find((device) => device.label === deviceLabel)?.deviceId;
 
-      // Filter out either video or audio from a given constraints and return a new media stream
-      if (constraints.video) {
-        stream = new MediaStream([mediaStream.getVideoTracks()[0]]);
-      }
+      if (ON_IOS_15_1) {
```

Comment:
this test should only be made when requesting a video stream, not an audio stream.

Reply:
and maybe it should be done before asking for permission for video

```js
const mediaStream = await navigator.mediaDevices.getUserMedia(constraints);